docs/dev/setup/server.rst

```diff
@@ -0,0 +1,291 @@
+  (see `Athena Service <#athena-service>`__).
+
+Run the server with Spring Boot and Spring profiles
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Artemis server should startup by running the main class
+``de.tum.in.www1.artemis.ArtemisApp`` using Spring Boot.
+
+.. note::
+    Artemis uses Spring profiles to segregate parts of the
+    application configuration and make it only available in certain
+    environments. For development purposes, the following program arguments
+    can be used to enable the ``dev`` profile and the profiles for JIRA,
+    Bitbucket and Bamboo:
+
+::
+
+   --spring.profiles.active=dev,bamboo,bitbucket,jira,artemis,scheduling
+
+If you use IntelliJ (Community or Ultimate) you can set the active
+profiles by
+
+* Choosing ``Run | Edit Configurations...``
+* Going to the ``Configuration Tab``
+* Expanding the ``Environment`` section to reveal ``VM Options`` and setting them to
+  ``-Dspring.profiles.active=dev,bamboo,bitbucket,jira,artemis,scheduling``
+
+Set Spring profiles with IntelliJ Ultimate
+""""""""""""""""""""""""""""""""""""""""""
+
+If you use IntelliJ Ultimate, add the following entry to the section
+``Active Profiles`` (within ``Spring Boot``) in the server run
+configuration:
+
+::
+
+   dev,bamboo,bitbucket,jira,artemis,scheduling
+
+Run the server with the command line (Gradle wrapper)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you want to run the application via the command line instead, make
+sure to pass the active profiles to the ``gradlew`` command like this:
+
+.. code:: bash
+
+   ./gradlew bootRun --args='--spring.profiles.active=dev,bamboo,bitbucket,jira,artemis,scheduling'
+
+As an alternative, you might want to use Jenkins and GitLab with an
+internal user management in Artemis, then you would use the profiles:
+
+.. code::
+
+   dev,jenkins,gitlab,artemis,scheduling
+
+
+Text Assessment Analytics is an internal analytics service used to gather data regarding the features of the text
+assessment process. Certain assessment events are tracked:
+
+1. Adding new feedback on a manually selected block
+2. Adding new feedback on an automatically selected block
+3. Deleting a feedback
+4. Editing/Discarding an automatically generated feedback
+5. Clicking the Submit button when assessing a text submission
+6. Clicking the Assess Next button when assessing a text submission
+
+These events are tracked by attaching a POST call to the respective DOM elements on the client side.
+The POST call accesses the **TextAssessmentEventResource** which then adds the events in its respective table.
+This feature is disabled by default. We can enable it by modifying the configuration in the file:
+``src/main/resources/config/application-artemis.yml`` like so:
+
+.. code:: yaml
+
+   info:
+      text-assessment-analytics-enabled: true
```

Comment:
This was moved to docs/admin/setup/text-assessment-analytics.rst

We should remove this occurrence